rest-service/manager_rest/rest/endpoint_mapper.py

```diff
@@ -99,7 +99,11 @@ def setup_resources(api):
         'ManagerConfig': 'config',
         'ManagerRestConfig': 'rest/config',
         'Agents': 'agents',
-        'AgentsName': 'agents/<string:name>'
+        'AgentsName': 'agents/<string:name>',
+        'Operations': 'operations',
+        'OperationsId': 'operations/<string:operation_id>',
+        'TasksGraphs': 'tasks_graphs',
+        'TasksGraphsId': 'tasks_graphs/<string:tasks_graph_id>'
```

Comment:
Trailing comma would be nice so that the next person to update this doesn't have it listed as them having changed one of your lines.